src/scss/_mixins.scss

```diff
@@ -69,11 +77,9 @@
 
 	.o-subs-card__copy-benefits {
 		@include oTypographyPadding($top: 3);
-		@include oColorsFor('o-subs-card');
```

Comment:
Won't this remove background colours?

Reply:
It would if I hadn't moved it to the element containing this one 😉

Reply:
Saw that after I made the comment 🤦‍♂️